Changelog.md

```diff
@@ -11,6 +11,7 @@ Compiler Features:
 Bugfixes:
 * Assembly-Json: Fix assembly json export to store jump types of operations in `jumpType` field instead of `value`.
 * TypeChecker: Convert parameters of function type to how they would be called for ``abi.encodeCall``.
+* View Pure Checker: Mark ``returndatasize`` and ``returndatacopy`` as view.
```

Comment:
Should be about marking them as not pure!

Reply:
How about now?

Reply:
Looks good!